test/test_blocks.py

```diff
@@ -911,5 +911,76 @@ def test_queue_enabled_for_fn():
     assert demo.queue_enabled_for_fn(1)
 
 
+@pytest.mark.asyncio
+async def test_queue_when_using_auth():
+    sleep_time = 5
+
+    async def say_hello(name):
+        await asyncio.sleep(sleep_time)
```

Comment:
I don't think we need to sleep for so long. The test takes quite a while to run right now.

Reply:
it is 1 sec and 3 times now